scripts/ep6.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think we want this file merged in, can you remove confirm and remove it if so?